go.mod

```diff
@@ -7,10 +7,10 @@ require (
 	github.com/coredns/coredns v1.11.1
 	github.com/miekg/dns v1.1.55
 	github.com/nginxinc/kubernetes-ingress v1.12.5
-	k8s.io/api v0.28.1
-	k8s.io/apimachinery v0.28.1
-	k8s.io/client-go v0.28.1
-	sigs.k8s.io/gateway-api v0.8.1
+	k8s.io/api v0.28.3
+	k8s.io/apimachinery v0.28.3
+	k8s.io/client-go v0.28.3
```

Comment:
```suggestion
	k8s.io/api v0.28.4
	k8s.io/apimachinery v0.28.4
	k8s.io/client-go v0.28.4
```

Reply:
Could also revert this and merge the existing PR https://github.com/ori-edge/k8s_gateway/pull/247

Reply:
Usually the dependabot will auto close if it sees the dependencies have been changed.

I ran

go mod tidy

To update the dependencies